### PR TITLE
Symfony4

### PR DESCRIPTION
**Pull request by lenardpalko**
Adding Symfony4 support.

During the SymfonyCon 2017 Fabien Potencier, announced that silex won't be maintained anymore and there were dependency version conflicts so I've removed the Silex support with this commit. 
In my opinion this could be a new release, but please let me know if you would like to still keep Silex support.

Thanks

#SymfonyConHackday2017

### CODE REVIEW

File: Silex/EkinoNewRelicServiceProvider.php [hunk 1]
```diff
@@ -1,117 +0,0 @@
-<?php
```
Comment (jderusse) [inline Silex/EkinoNewRelicServiceProvider.php]: Here you are removing the support of (the deprecated framework) silex wich is a BC break for this lib and should be done IMHO in a dedicated PR

**Comment by ChristianVermeulen** [comment]
@lenardpalko @jderusse Can we blow new life into this PR and get to a release? It would be nice to get rid of all the deprecation messages... 😎

**Comment by Nyholm** [comment]
Closed by #129
